assets/pages/digital-subscription-checkout/digitalSubscriptionCheckoutReducer.js

```diff
@@ -231,8 +231,11 @@ function initReducer(initialCountry: IsoCountry) {
     : Monthly;
   const { productPrices } = window.guardian;
 
+  const pathName = new URL(window.location).pathname.split('/');
+  const checkoutPathName: string = pathName.splice(pathName.findIndex(c => c === 'checkout')).pop();
```

Comment:
1. Should we do this? (vs a parameter or client side checking for id)
2. If we are to do this, should I extract this into a helper? So it has a more descriptive name